### PR TITLE
Proposal: OTLP Exporters Support for Configurable Export Behavior

### PR DESCRIPTION
**Pull request by huyan0**
This OTEP stems from discussions around spec [#731](https://github.com/open-telemetry/opentelemetry-specification/issues/731) and partially resolves  [#725](https://github.com/open-telemetry/opentelemetry-specification/issues/725).  The topic has been discussed at length in the Metrics SIG meeting, and the conclusion from those discussions is reflected here.

cc @huyan0 @alolita @jmacd @bogdandrutu

### CODE REVIEW

**Comment by bogdandrutu** [comment]
@huyan0 please fix markdown lint errors :) https://app.circleci.com/pipelines/github/open-telemetry/oteps/732/workflows/d88799f9-34eb-4763-ad4e-604ed3dd5197/jobs/927

**Comment by MrAlias** [comment]
> @huyan0 please fix markdown lint errors :) https://app.circleci.com/pipelines/github/open-telemetry/oteps/732/workflows/d88799f9-34eb-4763-ad4e-604ed3dd5197/jobs/927

And please remove the binary `DS_Store` files from the git history.

File: text/metrics/0131-otlp-export-behavior.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+## Prior art and alternatives
+
+A discussed solution is to convert deltas to cumulative in the Collector both as an agent and as a standalone service. However, supporting conversion in the Collector when it is a standalone service requires implementation of a routing mechanism across all Collector instances to ensure delta values of the same cumulative metric are aggregated by the same Collector instance. 
+
+## Open questions
+
+As stated in the previous section, delta to cumulative conversion in the Collector is needed to support Prometheus type backends. This may be necessary in the Collector in the future because the Collector may also accept metrics from other sources that report delta values. 
+
+The future implementation for this conversion in the Collector is still under discussion. There is a proposal is to add a [Metric Aggregation Processor](https://github.com/open-telemetry/opentelemetry-collector/issues/1422) in the Collector which recommends a solution.
```
Comment (MrAlias) [inline text/metrics/0131-otlp-export-behavior.md:39]: I thought the functional support of the collector to support cumulative -> delta was supposed to be a part of this proposal? Is that not the case?

Comment (jmacd) [reply text/metrics/0131-otlp-export-behavior.md:39]: I figure it belongs in the SDK specification, which I _promise_ to resume working on this week.

**Comment by jkwatson** [comment]
Is this a subset of the features described in OTEP 126? https://github.com/open-telemetry/oteps/blob/master/text/metrics/0126-Configurable-Metric-Aggregations.md

Or is this a requirement that OTLP exporters use the features in OTEP 126 and provide access via configuration?

**Comment by bogdandrutu** [comment]
I think it is a mix of both. But most importantly it says that defaults should be cumulative

**Comment by jmacd** [comment]
@jkwatson I support the idea that the Views configuration should be able to set the export strategy for OTLP. I see the goal of this OTEP as simply stating that (1) it should be a configuration option, (2) it defaults to cumulative. I'd say that an API to configure this is the MVP, and that a more wholistic Views API should include this feature as a configuration.

File: text/metrics/0131-otlp-export-behavior.md (hunk 1 above)
Comment (huyan0) [reply text/metrics/0131-otlp-export-behavior.md:39]: Hello, the reason I didn't add it is I thought we haven't concluded how the support for cumulative -> delta would look like if deltas are sent to different collector instances.

Comment (huyan0) [reply text/metrics/0131-otlp-export-behavior.md]: Hello, we haven't discussed in detail how the collector should support cumulative -> delta, and we thought having the OTLP exporter to be configurable would satisfy different backends. In the case of the collector taking metrics from other sources,  I think both cumulative -> delta and delta -> cumulative needs to be added.  I will add a sentence for that.

**Comment by huyan0** [comment]
I have removed the extra binary files, thanks : -)

**Pull request merged**